static/js/humhub/humhub.ui.modal.js

```diff
@@ -542,6 +543,17 @@ humhub.module('ui.modal', function (module, require, $) {
 
     var init = function () {
         module.global = Modal.instance('#globalModal');
+
+        if (pjax.isActive()) {
```

Comment:
Use this instead as mentioned  in https://github.com/humhub/humhub/issues/2950#issuecomment-770648113:

```
module.initOnPjaxLoad = true;
var init(pjax) {
    if(pjax) {
    ...
    }
```

This means that the init function will also be called after a pjax page load. An by the pjax argument of the init function you can check if the init call was triggered after pjax page load. You don't have to check if pjax is active.

Reply:
ok please review the commit https://github.com/humhub/humhub/pull/4812/commits/12d03a1522b489f340054c575345abc52b4bfc69

Reply:
Looks good, think this should be ok, pjax has a custom loader animation anyways also this case is not that common.